mm2src/common/common.rs

```diff
@@ -1080,8 +1080,18 @@ impl<Id> Default for PagingOptionsEnum<Id> {
 }
 
 #[inline(always)]
+#[cfg(not(feature = "for-tests"))]
 pub fn get_utc_timestamp() -> i64 { Utc::now().timestamp() }
 
+/// get_utc_timestamp for tests allowing to add some bias to 'now'
+#[cfg(feature = "for-tests")]
+pub fn get_utc_timestamp() -> i64 {
+    Utc::now().timestamp()
+        + std::env::var("TEST_TIMESTAMP_OFFSET")
+            .map(|s| s.as_str().parse::<i64>().unwrap_or_default())
+            .unwrap_or_default()
+}
+
```

Comment:
can we merge this functions as they seems very minimal and are unlikely to grow or maybe even change.

```rust
/// get_utc_timestamp for tests allowing to add some bias to 'now'
#[inline(always)]
pub fn get_utc_timestamp() -> i64 {
    #[cfg(feature = "for-tests")]
    return Utc::now().timestamp()
        + std::env::var("TEST_TIMESTAMP_OFFSET")
            .map(|s| s.as_str().parse::<i64>().unwrap_or_default())
            .unwrap_or_default();
    #[cfg(not(feature = "for-tests"))]
    return Utc::now().timestamp();
}


```

Reply:
I am okay with that, 
only we above discussed to make this as two separate functions, for normal calls and tests.

Reply:
Release runtime shouldn't have the offset logic, it's not safe.

Reply:
Then it should not be inline in this case, I guess

Reply:
It can be inlined. Compiler will still generate identical machine code compare to previous `get_utc_timestamp`.

Reply:
fixed in [6eaa3cf](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2304/commits/6eaa3cf8aa9998f2170cc6b13d75c54008effc33)